src/lib/operators/import.hpp

```diff
@@ -18,7 +15,9 @@ namespace hyrise {
 /*
  * This operator reads a file, creates a table from that input and adds it to the storage manager.
  * Supported file types are .tbl, .csv and Hyrise .bin files.
- * For .csv files, a CSV config is additionally required, which is commonly located in the <filename>.json file.
+ * For .csv files, a CSV config is additionally required (if no table with the same name exists as a template).
+ * The operator looks for this content in the file <filename>.json.
+ * If a table with this filename already exists, it is dropped (not appended as done in Postgres).
```

Comment:
I think it makes sense to match Postgres's behavior here (and add a test).

Reply:
I guess this is part of another comment, but I will leave this one open as a reminder to change this part as well.